improver/blending/weights.py

```diff
@@ -755,10 +755,13 @@ def triangular_weights(
             midpoint:
                 The centre point of the triangular function.
             width:
-                The width of the triangular function from the centre point.
+                The width from the triangle’s centre point, in units of the units
+                argument, which will determine the triangular weighting function
```

Comment:
> The width from the triangle’s centre point, in units of the **_plugin's_** units argument, which will determine the triangular weighting function....


This function doesn't have a units argument, so it's probably worth pointing the reader up to the plugin level.